services/api-server/src/simcore_service_api_server/clients/postgres.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import FastAPI
+from servicelib.fastapi.db_asyncpg_engine import close_db_connection, connect_to_db
+from servicelib.fastapi.lifespan_utils import LifespanOnStartupError
+from sqlalchemy.ext.asyncio import AsyncEngine
+
+from ..core.settings import ApplicationSettings
+
+
+class PostgresNotConfiguredError(LifespanOnStartupError):
+    msg_template = LifespanOnStartupError.msg_template + (
+        "Postgres settings are not configured. "
+        "Please check your application settings. "
+    )
+
+
+def get_engine(app: FastAPI) -> AsyncEngine:
+    assert app.state.engine  # nosec
+    engine: AsyncEngine = app.state.engine
+    return engine
+
+
+def setup_postgres(app: FastAPI):
```

Comment:
not brave enough to move to lifespans 🤣

Reply:
@GitHK i added it as a follow-up task. Check the description.